applications/asset_tracker/src/cloud_codec/cloud_codec.h

```diff
@@ -60,21 +60,31 @@ enum cloud_channel {
 	CLOUD_CHANNEL_LIGHT_BLUE,
 	/** The IR light level on the device. */
 	CLOUD_CHANNEL_LIGHT_IR,
+	/** The assisted GPS channel. */
+	CLOUD_CHANNEL_ASSISTED_GPS,
+
+	CLOUD_CHANNEL__TOTAL
 };
 
-#define CLOUD_CHANNEL_STR_GPS "GPS"
-#define CLOUD_CHANNEL_STR_FLIP "FLIP"
-#define CLOUD_CHANNEL_STR_BUTTON "BUTTON"
-#define CLOUD_CHANNEL_STR_TEMP "TEMP"
-#define CLOUD_CHANNEL_STR_HUMID "HUMID"
-#define CLOUD_CHANNEL_STR_AIR_PRESS "AIR_PRESS"
-#define CLOUD_CHANNEL_STR_AIR_QUAL "AIR_QUAL"
-#define CLOUD_CHANNEL_STR_LTE_LINK_RSRP "RSRP"
+#define CLOUD_CHANNEL_STR_GPS			"GPS"
```

Comment:
The alignment seems messed all below (checked in my editor, it's messed as well, though differently, so it's not a github issue). This is why I prefer to use spaces, or even avoid aligning at all (which helps to keep diffs smaller in case of any modifications).

Reply:
replaced with clang-format output (no alignment)